core/eval.go

```diff
@@ -820,11 +821,21 @@ func evalSTACKREFPEEK(args []string) []byte {
 // containing the new value of the key along with the operation that was performed on it.
 // Contains only one argument, the key to be watched.
 func evalQWATCH(args []string, c *Client) []byte {
-	if len(args) != 1 {
+	if len(args) < 2 {
 		return Encode(errors.New("ERR invalid number of arguments for `WATCH` command"), false)
```

Comment:
Error message should say `QWATCH` command instead of `WATCH` command.

Reply:
Done.